dashboard/app/controllers/curriculum_tracking_pixel_controller.rb

```diff
@@ -14,25 +14,45 @@ def index
       # For ease of querying we attempt to parse the url into meaningful chunks.
       # EXAMPLE:
       # /csf-18/pre-express/11/
+      # es-mx/csf-1718/coursec/10/'
       split_url = curriculum_page.split('/')
       # ["", "csf-18", "pre-express", "11"]
+      # ["", "es-mx", "csf-1718", "coursec", "10"]
 
+      non_en = split_url[1][2] == "-"
       if split_url.length > 1
-        # csf, csd, csp including version year, algebra or hoc
-        csx = split_url[1]
+        if non_en
+          locale = split_url[1]
+        else
+          locale = "en-us"
+          # csf, csd, csp including version year, algebra or hoc
+          csx = split_url[1]
+        end
       end
 
       if split_url.length > 2
-        # csf -> coursea, courseb, ..., pre-express, express
-        # csd/csp -> unit1, unit2, ...
-        # algebra -> courseA, courseB
-        # hoc -> plugged, unplugged
-        course_or_unit = split_url[2]
+        if non_en
+          csx = split_url[2]
+        else
+          # csf -> coursea, courseb, ..., pre-express, express
+          # csd/csp -> unit1, unit2, ...
+          # algebra -> courseA, courseB
+          # hoc -> plugged, unplugged
+          course_or_unit = split_url[2]
+        end
       end
 
       if split_url.length > 3
-        # lesson number, standards, vocab, resources
-        lesson = split_url[3]
+        if non_en
+          course_or_unit = split_url[3]
+        else
+          # lesson number, standards, vocab, resources
+          lesson = split_url[3]
```

Comment:
this part seems too hacky. a better way would be to pop the locale off the beginning of the array, and then treat the rest of the array the same regardless of english or non-english.